### PR TITLE
Srp workflows - Improvements working with Volume Components on a XPipeline Projects [Hold]

### PR DESCRIPTION
**Pull request by alex-vazquez**
---
### Purpose of this PR
https://jira.unity3d.com/browse/XPIPELINE-327
https://jira.unity3d.com/browse/XPIPELINE-328

Warning for Volume Components from another pipeline:
image

Add an option to hide not overrided parameters for a component
image
image

---
### Testing status
Describe what manual/automated tests were performed for this PR

---
### Comments to reviewers
Notes for the reviewers you have assigned.

### CODE REVIEW

File: com.unity.render-pipelines.core/Runtime/Volume/VolumeProfile.cs [hunk 1]
```diff
@@ -30,6 +32,40 @@ void OnEnable()
...
+            {
+                var currentPipelineType = currentPipeline.GetType();
+                foreach (var component in components)
+                {
+                    SetIsSupportedOnCurrentPipeline(component, currentPipelineType);
+                }
+            }
+        }
+
+        void SetIsSupportedOnCurrentPipeline(VolumeComponent component, Type currentPipelineType)
```
Comment (RSlysz) [inline com.unity.render-pipelines.core/Runtime/Volume/VolumeProfile.cs:59]: [Readability] Either rename `currentPipelineType` to `renderPipelineType` or make the method find who is the current one.

**Review by RSlysz — APPROVED** [review]
Looks good but please rename the parameter for better code readability.

File: com.unity.render-pipelines.core/Runtime/Volume/VolumeProfile.cs (hunk 1 above)
Comment (RSlysz) [reply com.unity.render-pipelines.core/Runtime/Volume/VolumeProfile.cs:59]: Same goes for method name. It should be `SetIsSupportedOnPipeline` if the method don't get who is the current one

**Review by TheoWong-pixel — APPROVED** [review]

File: com.unity.render-pipelines.core/Editor/Volume/VolumeComponentListEditor.cs [hunk 2]
```diff
@@ -308,7 +316,8 @@ void OnContextClick(Vector2 position, VolumeComponentEditor targetEditor, int id
                 menu.AddItem(EditorGUIUtility.TrTextContent("Show Additional Properties"), targetEditor.showAdditionalProperties, () => targetEditor.showAdditionalProperties ^= true);
             else
                 menu.AddDisabledItem(EditorGUIUtility.TrTextContent("Show Additional Properties"));
-            menu.AddItem(EditorGUIUtility.TrTextContent("Show All Additional Properties..."), false, () => CoreRenderPipelinePreferences.Open());
+            menu.AddItem(EditorGUIUtility.TrTextContent("Show Only Overrided Parameters"), targetEditor.showOnlyOverridedParameters, () => targetEditor.showOnlyOverridedParameters = !targetEditor.showOnlyOverridedParameters);
```
Comment (arttu-peltonen) [inline com.unity.render-pipelines.core/Editor/Volume/VolumeComponentListEditor.cs:319]: Proper form is "overridden", not "overrided". Replace at least user-facing text, preferably also variable names.

**Review by arttu-peltonen — CHANGES_REQUESTED** [review]
Happened to notice a typo in user-facing text, so jumped in to let you know :)

**Review by arttu-peltonen — CHANGES_REQUESTED** [review]
Sorry to nitpick but "overriden" is still not correct, "overridden" with two d's is correct spelling.

**Pull request closed without merging**